bin/hicAggregateContacts

```diff
@@ -288,6 +294,9 @@ def main(args):
     plt.savefig(args.outFileName.name, dpi=100, transparent=True)
     plt.close()
 
+    if args.outFileNameMatrix:
+        np.savetxt(args.outFileNameMatrix.name, chrom_matrix[chrom], '%0.5f', delimiter='\t')
+    
```

Comment:
I think that here you may have a problem because the file is rewritten in each loop. Thus, at the end the outFileNameMatrix file only contains the last chromosome.

One solution is to ask for a prefix and then save each chromosome as a separate file, adding to the file name the chromosome name.

Reply:
Ok I'll check that, for my tests I was using only one chromosome so I had no problem.